R/data.R

```diff
@@ -207,6 +207,57 @@ NULL
 #' }
 NULL
 
+#' Synthetic 2x2 Within-Subjects Experiment
+#' 
+#' Synthetic data from an experiment with two within-\code{Subject}s factors
+#' (\code{A} and \code{B}) having two levels each.
+#' 
+#' @name ElkinAB
+#' @docType data
+#' @format A data frame with 32 observations on the following 4 variables.
+#' \describe{ 
+#'      \item{S}{a factor with levels \code{"1"} .. \code{"8"}} 
```

Comment:
Can you make the factor labels for S something like "s1" ... "s8"? I don't like having factor labels that look like numerics, too often in simple `print()`ing the factor it is not numeric is not obvious